Notify/Notify/Notify/ViewModels/LoginPageViewModel.cs

```diff
@@ -30,51 +31,70 @@ public string Password
             get => m_Password;
             set => SetProperty(ref m_Password, value);
         }
-        
+
+        public bool RememberMe
+        {
+            get => m_RememberMe;
+            set
+            {
+                if (m_RememberMe != value)
```

Comment:
What's the scenario of this logic?

Reply:
It compares the new value (checked/unchecked) with the old value and only sets the property if the new value is different from the old value.
It's Necessary for storing the user credentials in the cache.